db/migrate/20160617123557_add_meta_fields_to_theming.rb

```diff
@@ -0,0 +1,7 @@
+class AddMetaFieldsToTheming < ActiveRecord::Migration
+  def change
+    add_column :pageflow_themings, :default_author, :string
+    add_column :pageflow_themings, :default_publisher, :string
+    add_column :pageflow_themings, :default_keywords, :string
```

Comment:
As an alternative, we can keep the fallback in the helper and display the default values from the global config as placeholders both in the theming form and the editor. That way defaults from the configuration can be used without having to migrate existing records. Soon as a theming has default values, those will be copied to the draft revision of new entries. Maybe this is a good compromise: Changing defaults in the initializer takes effect for all entries which do not have custom values. Changes there require a deploy and will probably invalidate caches anyway. Default values from the theming are copied to keep things simple for account managers. I'd remove the fallback from the `copy_defaults_to` in `theming.rb`, though.